### PR TITLE
Rakshit

### PR DESCRIPTION
**Pull request by v-raras**

### CODE REVIEW

File: CRM Solution Manager/CRM Solution/CrmConstants.cs [hunk 1]
```diff
@@ -58,6 +58,28 @@ public static string SolutionPackagerPath
             }
         }
 
+        /// <summary>
+        /// Gets solution packager path
```
Comment (amrullah08) [inline CRM Solution Manager/CRM Solution/CrmConstants.cs]: update comment

File: CRM Solution Manager/CRM Solution/CrmConstants.cs [hunk 2]
```diff
@@ -58,6 +58,28 @@ public static string SolutionPackagerPath
...
+        public static string MultilpleSolutionsImport
+        {
+            get
+            {
+                return ConfigurationManager.AppSettings["MultilpleSolutionsImport"];
+            }
+        }
+
+        /// <summary>
+        /// Gets solution packager path
```
Comment (amrullah08) [inline CRM Solution Manager/CRM Solution/CrmConstants.cs]: update comment

File: CRM Solution Manager/CRM Solution/CrmConstants.cs (hunk 1 above)
Comment (v-raras) [reply CRM Solution Manager/CRM Solution/CrmConstants.cs]: updated

File: CRM Solution Manager/CRM Solution/CrmConstants.cs (hunk 2 above)
Comment (v-raras) [reply CRM Solution Manager/CRM Solution/CrmConstants.cs]: updated

**Pull request merged**